CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ No date
 
 - Universal python build packages.
 - Use `pkg_resources` instead of `distutils` where available (allows for removal if `distutils` in python 3.12 - addresses [grahampugh/macadmin-scripts/issues/47](https://github.com/grahampugh/macadmin-scripts/issues/47)).
+- Allows the usage of spaces in `--workdir` and `--path`.
+- Fixed an issue with `--max-password-attempts=NN`.
```

Comment:
`--max-password-attempts` is new in 26.1, and have not yet released a full version, so this change note is not required.